### ISSUE TITLE
This is invaluable

### ISSUE THREAD

**Issue by arphex** [patois/RETracker#4]
I just wanted to say that this project can have an immense value.  Providing the hardware interface of the tracker to the community for developing open source software has the potential to be groundbreaking

Thank you

**Comment by blueneogeo** [comment]
Agreed, amazing work! Maybe the next step is to see if a more high level library can be built on top of this to trigger functions of the tracker.
